.github/workflows/build.yml

```diff
@@ -58,41 +74,70 @@ jobs:
           name: Tests report
           path: 'reports/jest-*.xml'
           reporter: jest-junit
+  
   package:
     name: Package and Push artifacts to Octopus
-    needs: test
+    needs: [build, test]
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
       - uses: actions/download-artifact@v2
         with:
           name: dist
           path: dist
+
       - name: Use Node.js
         uses: actions/setup-node@v2
         with:
           node-version: '16'
           cache: 'npm'
+
```

Comment:
Why install Go?

Reply:
I think this was a left over from early versions running node-prune to clean up. This looks to be taken care of ootb now, so I've removed it from here and the pack.ps1 file no longer calls go to run `node-prune`